syncer/syncer_test.go

```diff
@@ -1144,7 +1145,7 @@ func (s *testSyncerSuite) TestRun(c *C) {
 	}
 
 	mockStreamerProducer := &MockStreamProducer{s.generateEvents(events1, c)}
-	mockStreamer, err := mockStreamerProducer.generateStreamer(mysql.Position{})
+	mockStreamer, err := mockStreamerProducer.generateStreamer(binlog.Location{})
```

Comment:
I still prefer to use `NewLocation` instead of all `binlog.Location{}`, so access `location.GTIDSet` is valid.

Reply:
add NewLocation in e2e705e